CRM/Contact/DAO/SavedSearch.php

```diff
@@ -320,6 +322,10 @@ public static function &fields() {
           'entity' => 'SavedSearch',
           'bao' => 'CRM_Contact_BAO_SavedSearch',
           'localizable' => 0,
+          'html' => [
```

Comment:
Maybe just for my own understanding, but what's this doing?

Reply:
It's not directly related to exposing "description" but while I was in there I wanted to clean up the metadata a bit. Adding the label in metadata means SearchKit can then automatically pick it up (and I was able to remove it below and just pass `true` to get that label in the table).

Reply:
Makes sense, I thought it was probably unrelated, thanks.